test/parallel/test-net-listen-exclusive-random-ports.js

```diff
@@ -28,7 +28,7 @@ if (cluster.isMaster) {
 
   server.listen({
     port: 0,
-    exclusive: true
+    exclusive: true,
```

Comment:
Nit: please remove the comma here since it's an unrelated change. Thanks!

Reply:
The comma is still here after the update. As it's not related to this PR, could you please remove it?